src/iosource/pcap/Source.cc

```diff
@@ -176,7 +178,32 @@ void PcapSource::OpenOffline()
 	{
 	char errbuf[PCAP_ERRBUF_SIZE];
 
-	pd = pcap_open_offline(props.path.c_str(), errbuf);
+	FILE* f = nullptr;
+	if ( props.path == "-" )
+		{
+		f = stdin;
+		}
+	else
+		{
+		if ( ! (f = fopen(props.path.c_str(), "rb")) )
+			{
+			Error(util::fmt("unable to open %s: %s", props.path.c_str(), strerror(errno)));
+			return;
+			}
+
+		if ( BifConst::Pcap::bufsize_offline_bytes > 0 )
```

Comment:
`BifConst::Pcap::bufsize_offline_bytes` is an unsigned so this check seems redundant.

Reply:
An unsigned int can still be 0. But maybe `!= 0` would be better?